languages/csharp/exercises/concept/arrays/.meta/Example.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+public class BirdCount
+{
+    private int[] birdsPerDay;
+
+    public BirdCount(int[] birdsPerDay)
+    {
+        this.birdsPerDay = birdsPerDay;
+    }
+
+    public int Total()
+    {
+        var total = 0;
```

Comment:
https://docs.microsoft.com/fr-fr/dotnet/api/system.linq.enumerable.sum?view=netframework-4.8

Reply:
I know it can be done without it, but using `.Sum` is consistent with the idea of being `declarative` as says the text...

Reply:
I understand this suggestion, however, the example file should be the idiomatic way students should solve this exercise based on what whey know the will have learnt up until this exercise. As they won't know about LINQ but they do know about `foreach`, this is the idiomatic solution.

Reply:
I'll add documentation for this requirement later.